pytorch_lightning/core/lightning.py

```diff
@@ -403,6 +409,31 @@ def log_dict(
                 add_dataloader_idx=add_dataloader_idx
             )
 
+    @staticmethod
+    def __sync(
+        value: _METRIC,
+        sync_fn: Optional[Callable] = None,
+        sync_dist: bool = False,
+        sync_dist_op: Union[Any, str] = 'mean',
+        sync_dist_group: Optional[Any] = None,
+        device: torch.device = None,
+    ) -> _METRIC:
+        """Sync across workers when using distributed training"""
+        if not isinstance(value, (torch.Tensor, numbers.Number)):
+            return value
+
+        sync_fn = sync_fn or sync_ddp_if_available
+        dist_available = torch.distributed.is_available() and torch.distributed.is_initialized() or tpu_distributed()
```

Comment:
```suggestion
        dist_available = (torch.distributed.is_available() and torch.distributed.is_initialized()) or tpu_distributed()
```
just to be safe for future refactors?

Reply:
I don't get why. Why is it safer?

Reply:
its a nit. i was mixing up my associativity rules but parentheses will make the precedence clearer in case we need to add support for more comunication backends/devices later